src/components/Book.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Book = ({ id, title, category }) => (
+const Book = ({
+  id, title, category, removeBook, book,
+}) => (
   <tr>
     <td>{id}</td>
     <td>{title}</td>
     <td>{category}</td>
+    <td><button type="button" onClick={() => removeBook(book)}>Remove Book</button></td>
```

Comment:
Then edit here as appropriate. You may not need the `book` props as well.
```suggestion
    <td><button type="button" onClick={removeBook}>Remove Book</button></td>
```

Reply:
Sorry, it didn't work.